CONTRIBUTING.md

```diff
@@ -53,11 +53,9 @@ For features that you have a proposed solution for, please write an [RFC](./rfcs
 
 When submitting a pull request, please ensure your branch meets the following criteria:
 
-* It is a passing build (`gulp test --build` returns success).
 * The code is fully tested and has full coverage.
 * The code follows our coding guidelines (TBC).
 * The code does not introduce unnecessary dependencies (no jQuery!).
-* Ensure descriptive `.yml` file is added to the `./change/next` directory.
```

Comment:
Can we document in this file the commit strategy we are using and how that will ultimately form part of the release notes

https://github.com/Sage/carbon/blob/21b66cb43312635f58134844fad5281de03da8ab/rfcs/text/semantic-release.md#release-notes

Reply:
I'll get that added in.